pages/clubs.jsx

```diff
@@ -409,6 +409,7 @@ var ModalAddOrChangeYourClub = React.createClass({
              </div>
            : null}
           {this.renderValidationErrors()}
+          <p>We're full! Let us know if you'd like to be considered for the next round.</p>
```

Comment:
typographically speaking smart quotes please! 
![be-smart-dont-use-dumb-quote-marks](https://cloud.githubusercontent.com/assets/2492510/8706578/4b37b062-2ae5-11e5-8fda-3521adb94efe.jpg)
hehe :dancer: